templates/corporate/development-community.md

```diff
@@ -31,52 +35,71 @@ action in the development community.
-  example, avoid using a pronoun like “her” or “his” in sentences like
-  “Every developer should clean [their] keyboard at least once a
-  week.”
-
-## Getting help
-- When asking for help, provide the details needed for others to help
-  you. For example, include the **full traceback** in a [code
-  block](/help/code-blocks) (not a screenshot), a link to the code or
-  a draft PR you’re having trouble debugging, etc.
-- Ask questions on channels rather than DMing core contributors. You’ll
-  get answers faster since other people can help, and it makes it
-  possible for others to benefit from the discussion.
-- Don’t ask the same question in multiple places. Moderators read
-  every public channel, and make sure every question gets a reply.
-- Use @-mentions sparingly. Unlike IRC or Slack, Zulip’s topics make
-  it easy to see what you’re replying to, so you don’t need to mention
-  your conversation partner to make sure they will see or understand
-  your reply. Core contributors will often @-mention someone who asked
-  a question hours earlier to make sure they see the reply, but there
-  is generally no need to @-mention a core contributor unless you need
-  their timely attention.
+## How we communicate
+
+- Use **informal language**; there’s no need for titles like “Sir” or “Madam”.
+- Use **[gender-neutral
+  language](https://en.wikipedia.org/wiki/Gender-neutral_language)**. For
+  example:
+    - **Good**: Does anyone need a review on **their** PR?
+    - **Not good**: Does anyone need a review on **his** PR?
+- Aim to **communicate professionally**, using full sentences with correct spelling
+  and grammar.
+- Follow the community **[code of
+  conduct](https://zulip.readthedocs.io/en/latest/code-of-conduct.html)**.
+
+Read the [guide on how we
+communicate](https://zulip.readthedocs.io/en/latest/contributing/how-we-communicate.html)
+to learn more.
+
+## How to ask for help
+
+- Formulate a **clear question**, which includes an appropriate amount of context
+  and a specific request for help. Link to **relevant references** (e.g., GitHub
+  issues, Zulip's documentation, etc.).
+- When relevant, include the **full traceback** in a [code
+  block](/help/code-blocks) (not a screenshot).
+- If working on a code contribution, **try to solve your own problem first**,
+  including reading through relevant documentation and code. Identify and
+  describe the precise point on which you feel stuck.
+
+ Moderators read every public channel, and make sure that questions are
+ addressed. To avoid disrupting work in the community:
+
+- **Never ask the same question in multiple places**.
+- If you are new to the community, **default to never using @-mentions**.
+  Unnecessary mentions make it harder for Zulip's maintainers to see and respond
+  to urgent messages. You can use Zulip's [silent
+  mentions](https://zulip.com/help/mention-a-user-or-group#silently-mention-a-user)
+  to refer to a user without notifying them.
+
+## How to give feedback
```

Comment:
I wonder if this section would work better below "Where to post"?

Reply:
I wanted to put "Where to post" just above https://zulip.com/development-community/#where-do-i-send-my-message, which is further information on that topic (that is a useful reference, but is not required reading).